app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,38 @@
+class BookingsController < ApplicationController
+  before_action :logged_in_user
+  before_action :load_tour, only: %i(new save_booking)
+
+  def index; end
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new
+    @booking.tour_id = params[:tour_id].to_i
+    @booking.user_id = current_user.id
+    @booking.payment_id = 1
```

Comment:
tại sao gán bằng 1 vậy em.
mà chỗ này sao k dùng strong params em

Reply:
Chỗ này vì em chưa làm được thanh toán, nhưng trong database, chỗ table bookings có 1 trường là payment_id nối với bảng payments. nên em gán cho nó là 1, chứ để nil thì ko tạo booking được anh.

Reply:
sao không dùng strong params em

Reply:
chỗ này ban đầu em nghĩ chỉ cần truyền tour_id qua nên ko sử dụng strong params. Em đã fix lại rồi nha anh.